### PR TITLE
pin coverage>=4.0,<4.4

### PR DESCRIPTION
**Pull request by aconrad**
Should address https://github.com/benoitc/gunicorn/issues/1548.

### CODE REVIEW

**Comment by aconrad** [comment]
Hum... according to the Travis logs, the tests are still pulling `coverage==4.4.1` which is the buggy version, not sure why.

It works locally for me and installs `coverage==4.3.4`:

```
$ tox -e py27                                                                                                                                              [1:06:53]
py27 create: /Users/aconrad/work/gunicorn/.tox/py27
py27 installdeps: -rrequirements_test.txt, mock
py27 develop-inst: /Users/aconrad/work/gunicorn
py27 installed: coverage==4.3.4,funcsigs==1.0.2,-e git+git@github.com:aconrad/gunicorn.git@8c24d7522796b6663dd3c696216225305dba0b9b#egg=gunicorn,mock==2.0.0,pbr==3.1.1,py==1.4.34,pytest==3.0.5,pytest-cov==2.4.0,six==1.10.0
...
```

Ideas?

File: requirements_test.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
+coverage>=4.0,<4.4
```
Comment (berkerpeksag) [inline requirements_test.txt]: Can you add a TODO comment so we can find the reason behind the change without digging commit logs in the future?

Comment (aconrad) [inline requirements_test.txt]: Sure! 740395a

**Comment by berkerpeksag** [comment]
I think that's because Travis didn't recreate the venv in that run. Can you please add `-r` to https://github.com/benoitc/gunicorn/blob/master/.travis.yml#L25

**Comment by berkerpeksag** [comment]
I think we might just bitten by https://github.com/tox-dev/tox/issues/149.

**Comment by aconrad** [comment]
Ok, `--recreate` did the trick apparently. Now coverage reports work but it all looks a little garbage because Couverture has been receiving bogus coverage reports. But if we merge this PR then it will generate a proper coverage report for `master` and then subsequent PRs should start seeing proper coverage reports again as long as they are generated with a working coverage version.

**Comment by berkerpeksag** [comment]
Done, thanks! I'm going to rerun all PRs once a proper report for master is created.

**Comment by aconrad** [comment]
Thanks @berkerpeksag! I'm going to bed now, I'll check-in again tomorrow. If it works you can close #1548, otherwise just let me know and I'll look into it.

**Comment by tilgovi** [comment]
Thank you @aconrad!

**Pull request merged**